### ISSUE TITLE
Two instances of 1 RHEA mapping to two terms found in ontology: first instance RHEA:46332

### ISSUE THREAD

**Issue by hdrabkin** [geneontology/go-ontology#23540]
Found while doing a PR for a new term, I was blocked by these two uses of 1 GO term for more than one GO id. I put a temp fix into these so I could get my NTR through. These did not come from my work for the pull, so were apparently there prior. This should have been flagged previously (one RHEA cannot be used for two GO terms as an exactMatch xref.) Jim has been notified. 
Now to the two RHEA ids

RHEA:46332,RHEA:,http://purl.obolibrary.org/obo/GO_0050184;http://purl.obolibrary.org/obo/GO_0102855
RHEA:30059,RHEA:,http://purl.obolibrary.org/obo/GO_1990663;http://purl.obolibrary.org/obo/GO_0004158

1 .  The following relates to the first, RHEA:46332, mapping to both GO:0050184 and GO:0102855. I will address the second in a separate ticket (#23541).
GO_0050184 phosphatidylcholine desaturase activity
DEF: Catalysis of the reaction: 1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + NAD(+) = 1-acyl-2-linoleoyl-sn-glycero-3-phosphocholine + H(+) + NADH.
Substrate has synonym  1-acyl-2-(9Z)-octadecenoyl-sn-glycero-3-phosphocholine in CHEBI:

this term used EC:1.14.19.22 as xref (which maps to RHEA:46332)
BUT: The RHEA reaction does not have NAD as substrate, NOR does the EC: 
I think both should be removed from this term  
During my PR for my new term, I temporarily made the RHEA a relatedMarch exref.
However,  the Metacyc mapped to this go term: PHOSPHATIDYLCHOLINE-DESATURASE-RXN
does NOT use NAD, yet references the EC (which does not)

This term has 1 IEA anotation to an Arabidopsis thaliana gene FAD2 (fatty acid desaturase 2)  based on the EC mapping

The other term  (GO:0102855) does not have NAD as a substrate. 

GO_0102855 1-18:1-2-18:2-phosphatidylcholine desaturase activity
DEF: Catalysis of the reaction: 1-18:1-2-18:2-phosphatidylcholine + O2 + a reduced electron acceptor = 1-18:2-2-18:2-sn-glycerol-3-phosphocholine + 2 H2O + an oxidized electron acceptor.
EC:1.14.19.22;   Acyl-lipid omega-6 desaturase (cytochrome b5)  
DEF: An oleoyl-[glycerolipid] + 2 ferrocytochrome b5 + O(2) + 2 H(+) <=> a linoleoyl-[glycerolipid] + 2 ferricytochrome b5 + 2 H(2)O

Because of the EC, the term to RHEA:4633 a (9Z)-octadecenoyl-containing glycerolipid + 2 Fe(II)-[cytochrome b5] + 2 H+ + O2 = a (9Z,12Z)-octadecadienoyl-containing glycerolipid + 2 Fe(III)-[cytochrome b5] + 2 H2O

This term has 1 IEA annotation to an Arabidopsis thaliana gene FAD2 (fatty acid desaturase 2) based on the IEA mapping.

Two points: 
1. should BOTH terms be obsoleted. 
2  @amorgat : is '1-18:1-2-18:2-phosphatidylcholine' related to a (9Z)-octadecenoyl-containing glycerolipid ? or was the EC added incorrectly? In which case this term would have no vaild Def refs. ;

**Comment by amorgat** [comment]
Hi Harold, I will discuss your issue with @kaxelsen

**Comment by kaxelsen** [comment]
EC 1.14.99.22 was created as EC 1.3.1.35 with the reaction 1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + NAD(+) = 1-acyl-2-linoleoyl-sn-glycero-3-phosphocholine + NADH.
i.e. the reaction found in GO_0050184. The EC number was in 2015 transferred to EC 1.14.19.22 as it turned out it was using dioxygen and cytochrome b5 to oxidase the acyl chain. 
The new IUBMB reaction is 
an oleoyl-[glycerolipid] + 2 ferrocytochrome b5 + O2 + 2 H+ = a linoleoyl-[glycerolipid] + 2 ferricytochrome b5 + 2 H2O = RHEA:46322.

The reaction used in GO_0050184, 1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + NAD(+) = 1-acyl-2-linoleoyl-sn-glycero-3-phosphocholine + H(+) + NADH is partially a child of RHEA:46322 as the substrate and products are children of the substrate/product pair of RHEA:64322. The reaction deos not correspond to PHOSPHATIDYLCHOLINE-DESATURASE-RXN as this reaction has been updated to use dioxygen and cytochrome b5. We do not have a reaction corresponding to the MetaCyc reaction in Rhea.

The reaction used in GO_0102855 is, if updated to use cytochrome b5, a child of PHOSPHATIDYLCHOLINE-DESATURASE-RXN. We do not have a corresponding reaction in Rhea.

**Comment by hdrabkin** [comment]
We will merge all instances of use of EC:1.14.19.22 into GO:0050184; make xrefs narrow matches.

**Comment by pgaudet** [comment]
@kaxelsen looks like you have a typo, it's RHEA:46332 NOT RHEA:46322

**Comment by hdrabkin** [comment]
There were merged into GO:0050184
+alt_id: GO:0102855
+alt_id: GO:0102857
+alt_id: GO:0102860
+alt_id: GO:0102861
+alt_id: GO:0102871
+def: "Catalysis of the reaction: An oleoyl-[glycerolipid] + 2 ferrocytochrome b5 + O(2) + 2 H(+) = a linoleoyl-[glycerolipid] + 2 ferricytochrome b5 + 2 H(2)O." [GOC:curators]
+synonym: "1-16:0-2-18:1-phosphatidylcholine desaturase activity" NARROW []
+synonym: "1-18:1-2-18:2-phosphatidylcholine desaturase activity" NARROW []
+synonym: "1-18:1-2-18:2-phosphatidylcholine synthase activity" NARROW []
+synonym: "1-18:1-2-18:3-phosphatidylcholinedesaturase activity" NARROW []
+synonym: "1-18:2-2-18:1-phosphatidylcholine desaturase activity (SN2-18:2 forming)" NARROW []

**Comment by kaxelsen** [comment]
> 

Yes, but I think the message was clear.
